src/components/UI/Inputs/ViewEditName.tsx

```diff
@@ -67,11 +73,17 @@ class ViewAndEditName extends Component<
     return null;
   };
 
-  static validate(value: string): { valid: boolean; error: string } {
+  static validate(
+    value: string,
+    variant: string
```

Comment:
Is this a `string` on purpose? Or should it be `ViewAndEditNameVariant`?

Reply:
Nope that should be `ViewAndEditNameVariant`!